changelog/7683.improvement.rst

```diff
@@ -0,0 +1 @@
+The formerly optional ``pygments`` dependency is now required, causing output always to be source-highlighted (unless disabled).
```

Comment:
How can it be disabled again? Probably worth noting that ("unless disabled via...").